tests/integration/nfs_test.go

```diff
@@ -105,7 +105,7 @@ func (suite *NfsSuite) TestNfsServerInstallation() {
 		"1 rook-nfs-operator must be in Running state")
 
 	// verify nfs server instances are running OK
-	assert.True(suite.T(), suite.k8shelper.CheckPodCountAndState("rook-nfs", suite.namespace, suite.instanceCount, "Running"),
+	assert.True(suite.T(), suite.k8shelper.CheckPodCountAndState(suite.namespace, suite.namespace, suite.instanceCount, "Running"),
```

Comment:
can we ensure we have an issue opened for adding an integration test for this dynamic provisioning flow?

Reply:
sure

Reply:
yeah opened the issue here https://github.com/rook/rook/issues/3038